packages/backend-lib/src/userEvents.ts

```diff
@@ -136,6 +137,7 @@ export async function findManyEvents({
   // unix timestamp units ms
   startDate?: number;
   endDate?: number;
+  searchTerm?: string;
```

Comment:
Doing local testing, it doesn't seem that a `searchTerm` value is being passed here when I type into the search bar.